api/types/types_drivers_executor.go

```diff
@@ -46,6 +46,18 @@ const (
 
 	// LSXCmdMounts is the command for getting a list of mount info objects.
 	LSXCmdMounts = "mounts"
+
+	// LSXCmdVolumeCreate is the command for creating a volume.
+	LSXCmdVolumeCreate = "volumeCreate"
+
+	// LSXCmdVolumeRemove is the command for removing a volume
+	LSXCmdVolumeRemove = "volumeRemove"
+
+	// LSXCmdVolumeAttach is the command for attaching a volume.
+	LSXCmdVolumeAttach = "volumeAttach"
+
+	// LSXCmdVolumeDeatch is the command for detaching a volume.
+	LSXCmdVolumeDeatch = "volumeDetach"
```

Comment:
Hi @codenrhoden,

Sorry, this is labelled a work-in-progress, but I only updated the situation in Slack where I mentioned the PR is incomplete.